File: MyBucketListApp/app/src/main/java/com/example/mybucketlistapp/MainActivity.java
package com.example.mybucketlistapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;
import androidx.recyclerview.widget.RecyclerView;

import android.content.Intent;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    CardView toDosCard;
    CardView toGoesCard;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        toDosCard = findViewById(R.id.card_view_to_do);
        toGoesCard = findViewById(R.id.card_view_to_go);

        toDosCard.setOnClickListener(view -> {
            Intent toDoActivityIntent = new Intent(MainActivity.this, ToDoActivity.class);

            startActivity(toDoActivityIntent);
        });

        toGoesCard.setOnClickListener(view -> {
            Intent toGoActivityIntent = new Intent(MainActivity.this, ToGoActivity.class);

            startActivity(toGoActivityIntent);
        });

    }
}

File: HelpFileSendData/app/src/main/java/com/example/helpfilesenddata/MainActivity.kt
package com.example.helpfilesenddata



import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {
    private lateinit var etId: EditText
    private lateinit var etName: EditText
    private lateinit var etRoll: EditText
    private lateinit var btnSend: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        etId = findViewById(R.id.etId)
        etName = findViewById(R.id.etName)
        etRoll = findViewById(R.id.etRoll)
        btnSend = findViewById(R.id.btnSend)

        btnSend.setOnClickListener {

            val bundle = Bundle()
            bundle.putString("id", etId.text.toString())
            bundle.putString("name", etName.text.toString())
            bundle.putString("roll", etRoll.text.toString())

            val intent = Intent(this, SecondActivity::class.java)
            intent.putExtras(bundle)
            startActivity(intent)
        }
    }
}
